xmrstak/cli/cli-miner.cpp

```diff
@@ -230,10 +245,30 @@ int main(int argc, char *argv[])
 		;
 		configEditor configTpl{};
 		configTpl.set(std::string(tpl));
+		std::cout<<"Please enter:"<<std::endl;
+		auto& currency = params::inst().currency;
+		if(currency.empty())
```

Comment:
This if is never executed in my testing. After `rm *.txt; ./xmr-stak` the miner goes on to mine aeon without prompting.

Reply:
Just to make sure checked cmake output -> `-- Set miner currency to 'monero' and 'aeon'`

Reply:
well spotted: I inserted two logic bugs during the renaming

- fix with the last commit